Gems/ROS2/Code/Include/ROS2/Sensor/SensorHelper.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Component/Component.h>
+
+namespace ROS2
+{
+    //! Function to check if the component is a sensor component
+    //! @param component Component to check
+    //! @return True if the component is a sensor component
+    bool IsComponentROS2Sensor(const AZ::Component* component);
+
+    //! Function to get Ids of all sensors attached to the entity
+    //! @param entityId Id of the entity
+    //! @return Vector of EntityComponentIdPair of all sensors attached to the entity
+    AZStd::vector<AZ::EntityComponentIdPair> GetSensorsForEntity(const AZ::EntityId& entityId);
```

Comment:
I think `AZStd::vector<AZ::ComponentId>` would be better return type from this API, because each component will belong to the same entity, whose entityId is already known by the caller

Reply:
Disagree, since I want to have a type that can be passed directly to SensorConfigurationRequestBus. Using only ComponentId would cause some boilerplate code to be needed by API consumer.

Reply:
I second @MateuszWasilewski in this matter. 
I understand your point in making interfaces easier to use with the bus, but it makes the API weird. When asking for the list of sensors for the given entity, I expect nothing but the list of sensors. I know the _id_ of the entity at this point and I don't need to get it back duplicated with every sensor _id_.

Reply:
I agree that a raw Component vector would be a better fit, but in my opinion, the proposed approach is also acceptable. 

A vector of components is more semantically correct, but this one has some valid quality-of-life improvement.